assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -1000,9 +1000,11 @@ def __init__(self, n_retry, backoff_factor):
 
         retry_strategy = Retry(
             total=n_retry,
+            status=n_retry,
             status_forcelist=[104, 408, 409, 424, 429, 500, 502, 503, 504],
```

Comment:
does it make sense to retry on 503?

Reply:
I think so--
<img width="718" alt="image" src="https://github.com/Azure/azureml-assets/assets/42190563/1bc18c58-20fe-4324-a078-22467ddbc626">
I'm not the most well versed in http status codes but if the service is temporaryily down, we could retry in the hopes that it is temporary. Also other instance of retry in azureml-assets repo, retries on 503

Reply:
got it. it may require a longer backoff though.

Reply:
I was thinking that as well, I'm going to check this one in since i have approvals and have created this WI to come back to this : [Bug 2797021](https://msdata.visualstudio.com/Vienna/_workitems/edit/2797021): [Monitoring][P2]improve backoff strategy